lab-dan/data/1234-test-obj.json

```diff
@@ -0,0 +1 @@
+{"id":"1234-test-obj","creationDate":"2016-12-11T06:06:33.156Z","name":"Test","breed":"Shiba"}
```

Comment:
Go ahead and gitignore your data directory. No need for those files in the repo at this point.